### ISSUE TITLE
Default MeshAngularDeflection (28.50) too coarse

### ISSUE THREAD

**Issue by pmjdebruijn** [FreeCAD/FreeCAD#18789]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

Currently the Part/PartDesign workbench's MeshAngularDeflection parameter defaults to 28.50 degrees, which results in round shapes that are obviously not round.

After some practical testing, it seems like 10 degrees would be a good sweet spot, as round objects look nearly round, without generating exorbitant amounts of rendered geometry.

### Full version info

```shell
1.0.0+41~gd9e8fe0d7d
```

### Subproject(s) affected?

PartDesign

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by FlachyJoe** [comment]
I disagree, for small parts the value can decrease but for large assembly it's a good compromise.

Related: #18775 #18378
